### PR TITLE
Rebranding-blog

### PR DESCRIPTION
**Pull request by jilaga**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -49,4 +49,111 @@ document.addEventListener('DOMContentLoaded', function() {
 			e.currentTarget.parentElement.parentElement.querySelector(".expertise-detail").classList.toggle('is-hidden');
 		});
 	}
-});
\ No newline at end of file
+});
```
Comment (bebatut) [inline js/scripts.js:52]: Could you revert the changes on this file? Thanks

File: css/custom.scss [hunk 2]
```diff
@@ -338,4 +338,5 @@ $link-active: $link-hover;
   margin-right: auto;
   margin-top: 25px;
   margin-bottom: 20px;
+  cursor: pointer;
```
Comment (bebatut) [inline css/custom.scss:341]: Could you revert the changes on this file? Thanks

File: _posts/2023-01-31-rebranding-so-far.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  name: Photo by Milad Fakurian on Unsplash
+  url: https://unsplash.com/photos/0n1pmev3w2A
+---
+
+OLS is undergoing a rebranding project to make it more inclusive and accessible to researchers from various fields of study. The project is led by a design intern from Outreachy, who brings a fresh outlook to the process. So far, we have  conducted research and documentation to understand OLS's ideal targets and collaborators, and the current brand positioning. The biggest challenge faced is generating a new brand name. OLS is reaching out to its community for support and to vote for their preferred brand name choice from a shortlisted two.
+
+> Hello! I am Jilaga Nneoma. As the design intern at OLS, I am thrilled to be playing a key role in this rebranding project. My passion for design, combined with my Outreachy Internship, has given me the opportunity to offer a unique perspective and contribute new ideas. Working with OLS on this exciting project is a true honor and I am eager to share with you what we have accomplished so far. 
+
+
+## **Rebranding OLS:** The Journey so far...
```
Comment (bebatut) [inline _posts/2023-01-31-rebranding-so-far.md:17]: Can you remove the bold in all section titles? Thanks

File: _posts/2023-01-31-rebranding-so-far.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Rebranding so far...
+authors:
+- Jilaga
```
Comment (bebatut) [inline _posts/2023-01-31-rebranding-so-far.md:5]: You need to add yourself to the `_data/people.yaml` file

File: _posts/2023-01-31-rebranding-so-far.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+After much consideration and review, we have narrowed down our list of potential new brand names to just two options that capture the essence of OLS.
+This is where you come in! We are now eager to hear from you on which option you prefer.
+
+We value the power of community, openness and collaboration and would love to demonstrate that by  reaching out to our community for help. Your feedback is crucial in helping us make this important decision, and by including your input in this process, we can ensure that the new name accurately reflects the diversity of our community and our mission to nurture and grow.
+
+To participate in the voting process, simply follow the [link](https://voteupapp.com/shared/B7yJUuS6F) provided. The survey will take just a few minutes to complete and you can use your preferred alias in your response. Your input is incredibly valuable to us, and we look forward to seeing which of the select name resonates most with our community.
+
+[vote here](https://voteupapp.com/shared/B7yJUuS6F)
```
Comment (bebatut) [inline _posts/2023-01-31-rebranding-so-far.md:73]: You could maybe make it as a button. See the `index.md` for an example

**Review by bebatut — COMMENTED** [review]
Thanks @jilaga for the post

I put comments directly in the files. 

I just think about an extra comment. Could you put in bold some keywords in the text? It would help people reading and getting the key messages in the paragraph. Thanks

**Comment by jilaga** [comment]
> Thanks @jilaga for the post
> 
> I put comments directly in the files.
> 
> I just think about an extra comment. Could you put in bold some keywords in the text? It would help people reading and getting the key messages in the paragraph. Thanks

Awesome! I would resolve and treat them accordingly. Thank you

File: _data/people.yaml [hunk 6]
```diff
@@ -5838,3 +5838,19 @@ zenita-milla-luthfiya:
...
+  - Brand design
+  - User experience design
+  - User interface design
+  - Version Control
+  - User research
+  first-name: Nneoma
+  last-name: Jilaga
+  pronouns: She/her 
+  title: Brand design intern 
+  twitter: https://twitter.com/nohmaah
```
Comment (bebatut) [inline _data/people.yaml:5856]:
```suggestion
  twitter: nohmaah
```

**Comment by jilaga** [comment]
@yochannah here it is

**Comment by yochannah** [comment]
@jilaga  make sure to close this once your other PR is ready!

**Comment by jilaga** [comment]
Okay! Thank you 😊
